runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineOptions.java

```diff
@@ -44,4 +47,18 @@
   @Default.Long(1000)
   Long getBatchIntervalMillis();
   void setBatchIntervalMillis(Long batchInterval);
+
+  @Override
+  @Default.String("spark dataflow pipeline job")
```

Comment:
This was removed a while ago since it is inherited from `ApplicationNameOptions`